backend/src/jobs_server/models.py

```diff
@@ -103,3 +104,20 @@ def from_managed_workload(cls, workload: "ManagedWorkload") -> Self:
             spec=workload.spec,
             kueue_status=workload.status,
         )
+
+
+class LogsParams(BaseModel):
```

Comment:
This would presumably need to be kept in sync with `jobby logs`'s argument names and types?

Reply:
Yes, though that was also true for the param raw param names in the FastAPI endpoint.
The main rationale to add this here was to allow for the validation of `tail >= -1`.